openfecwebapp/templates/macros/filters/date.html

```diff
@@ -46,7 +46,6 @@
   </div>
   <div class="filter js-filter" id="{{ name }}" data-filter="date" data-name="{{ name }}" data-validate="true">
     <fieldset>
-    <legend class="label">{{ label }}</legend>
```

Comment:
Now that I think about it, since it's a form, should we still keep the legend but hide the text for accessibility purposes?